src/kOS.Safe/Exceptions/KOSBadJumpException.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace kOS.Safe.Exceptions
+{
+    public class KOSBadJumpException : Exception
+    {
+        public KOSBadJumpException(int destination, string message):
+            base( String.Format("Can't jump to instruction {0}.  No opcode there: {1}: {2} ",
+                                destination,
+                                message,
+                                "If you see this message, something is broken about how this program got compiled."))
+        {
+        }
+
+        public virtual string VerboseMessage
+        {
+            get { return base.Message; }
+        }
+
+        public virtual string HelpURL
+        {
+            get { return string.Empty; }
+        }
+    }
+}
```

Comment:
This new exception is just here so that if the problem, or a similar one appears again, the attempt to jump to instruction -1 should throw this error instead of re-running the program.